src/Symfony/Component/AssetMapper/Command/AuditCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Symfony\Component\AssetMapper\Command;
+
+use Symfony\Component\AssetMapper\ImportMap\ImportMapAuditor;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(name: 'importmap:audit', description: 'Checks for security vulnerability advisories for dependencies.')]
+class AuditCommand extends Command
+{
+    private const SEVERITY_COLORS = [
+        'info' => 'default',
+        'low' => 'default',
+        'moderate' => 'yellow',
+        'high' => 'red',
+        'critical' => 'red',
+    ];
+
+    public function __construct(
+        private readonly ImportMapAuditor $importMapAuditor,
+    ) {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $audit = $this->importMapAuditor->audit();
+
+        if ([] === $audit) {
+            $io->info('No packages to audit.');
```

Comment:
instead of always outputting text, I think we should support selecting the output format. The audit command looks like a good candidate for usage in scripted contexts.
For reference, `composer audit` supports a JSON output.

Reply:
I added a `format` option which supports "txt" or "json" for now.